DXClient.sln

```diff
@@ -16,714 +16,234 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{0F60E4A3-55C8-4C3F-BFC0-27C597433E8A}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
+		Directory.Build.props = Directory.Build.props
+		build\Framework.props = build\Framework.props
```

Comment:
Can you make it work by only making these changes to the solution file? All other changes to the solution file I would like to revert. Without that specific solution configuration it is not possible to debug for example the code for the combination TS + XNA + Debug + x86, DX + GL + Release + x64, etc. in VS.

Reply:
There are still some changes that should be reverted. When you open Configuration Manager in VS and select any XNA configuration as active configuration in combination with any active platform other than x86 the Build checkbox should be disabled.
It should be safe to just copy paste the contents of the old file and add the above 2 lines again I believe.